_docs/10-support.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: splash
+title: "Support"
+permalink: /docs/support/
+hidden: true
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/201106-CORP-BNR-Design-Centers-FPGAs-and-plds-Banner-2880x280.jpg
+excerpt: "Welcome to the Support page."
+---
+
+For general Libero SoC issues please contact [Microchip Technical Support](http://soc.microsemi.com/Portal/DPortal.aspx?v=1).
+
+For Linux and Bare metal issues please use the [RISC-V Technical Support Forum](https://exchange.riscv.org/).
```

Comment:
Bare metal issues -> bare metal software issues

Reply:
Agreed, will make this change now.